app/models/ckey2bibframe.rb

```diff
@@ -12,7 +12,8 @@ def convert
     catalogdump = "echo #{ckey} | #{Settings.symphony_catalogdump}"
 
     jar = 'lib/xform-marc21-to-xml-jar-with-dependencies.jar'
-    java_command = "java -Djava.security.egd=file:/dev/../dev/urandom -cp #{jar} edu.stanford.MarcToXMLStream"
+    java_command = "java -Djava.security.egd=file:/dev/../dev/urandom -cp #{jar} \
+                    edu.stanford.MarcToXMLStream -p config/settings/server.conf"
```

Comment:
Just a suggestion - maybe it's easier to read in separate lines/vars? i.e.
```ruby
jar = 'lib/xform-marc21-to-xml-jar-with-dependencies.jar'
opts = '-Djava.security.egd=file:/dev/../dev/urandom'
convert = 'edu.stanford.MarcToXMLStream -p config/settings/server.conf'
java_command = "java #{opts} -cp #{jar} #{convert}"
```
Let me know if you want to use that, otherwise it's good to go.